scripts/functions.js

```diff
@@ -1,3 +1,55 @@
 'use strict'
 
+function letras(cadena) { return cadena.length; }
 
+function palabras(cadena){ return cadena.split(' ').length; }
+
+function maysc(cadena) { return cadena.toUpperCase(); }
+
+/*
+
+ ^\w: toma el primer caracter de la cadena.
+ |: condicional OR.
+ \s\w: toma el primer caracter después de un espacio en blanco.
+ (): captura el patrón.
+ g: empareja las ocurrencias.
+
+*/
+function titulo(cadena) { return cadena.replace(/(^\w|\s\w)/g, m => m.toUpperCase()); } 
+
+function letrasReves(cadena) { 
+    
+    var cadenaInvertida = '';   var p = cadena.length;
+
+    while (p >= 0) {
+
+        cadenaInvertida = cadenaInvertida + cadena.charAt(p);
+        p--;
+    }
```

Comment:
Dije que no usarais for pero no para que uséis while...
Has visto que es mucho más sencillo con el reverse de arrrays, ¿no? Como has hecho en el siguiente...